src/Database/Esqueleto/PostgreSQL.hs

```diff
@@ -363,3 +371,75 @@ filterWhere aggExpr clauseExpr = ERaw noMeta $ \_ info ->
     in ( aggBuilder <> " FILTER (WHERE " <> clauseBuilder <> ")"
        , aggValues <> clauseValues
        )
+
+newtype PgValuesExprs a = PgValuesExprs { unPgValuesExprs :: NE.NonEmpty a }
```

Comment:
Is this a Postgres specific feature?

[This article](https://modern-sql.com/feature/values) seems to indicate that it is a `SQL-92` feature, and should be supported by all SQL databases.

[MySQL supports `VALUES`](https://dev.mysql.com/doc/refman/8.0/en/values.html), as does [SQLite](https://www.sqlite.org/lang_select.html). So I think we can move this into `esqueleto` proper.

Reply:
Ah sorry: my bad! Thanks for the links.

Sure, I can do that, but looks like other vendors requires different syntax, so I need to address that then :)

I will try to do that.

Reply:
Oh, if they require different syntax, then let's not bother. The current solution is to keep anything that varies in these separate modules.

Reply:
In some basic detail:
- MySql wants keyword `ROW(..)` for one row of values;
- SqLite requires some other placement of commas as far as I understand.. but worth to play a bit more.

I think consistency of API for vendors matters, so I am ready to try to cover other vendors as well.
But if you allow to go Postgres API earlier than other vendors - it would be awesome!

Otherwise - I could add necessary extensions for other vendors right here and ping you when it's ready :)

Reply:
`esqueleto` doesn't currently do anything to differentiate the database provider - it's all `Sqlbackend` as far as we're concerned. So I'd rather not try and do anything clever here and just support PG directly. If someone else wants MySQL support or Sqlite, then we can add those in vendor-specific modules, and if we really want to unify it, we can try and do that.

Reply:
Yep, understood: I also meant similar extensions in vendor-specific modules (like here) without unification (which probably also will present some code duplication but not a big deal I think).